frontend/packages/noobaa-storage-plugin/src/utils.ts

```diff
@@ -27,3 +27,16 @@ export const getPhase = (obj: K8sResourceKind): string => {
 export const isBound = (obj: K8sResourceKind): boolean => getPhase(obj) === 'Bound';
 
 export const getSCProvisioner = (obj: StorageClass) => obj.provisioner;
+
+export const isFunctionThenApply = (fn: any) => (args: string) =>
```

Comment:
```suggestion
export const isFunctionThenApply = (fn: Function) => (args: string) =>
```

Reply:
fn can be anything. Hence we are testing if it's a function and then applying the arguments.